doc/guides/west/release-notes.rst

```diff
@@ -1,6 +1,91 @@
 West Release Notes
 ##################
 
+v0.6.x
+******
+
+- No separate bootstrapper
+
+  In west v0.5.x, the program was split into two components, a bootstrapper and
```

Comment:
Maybe you could link to the 1.14 doc here, where all the concepts described in this paragraph are described

Reply:
Thanks for bringing up the v1.14 documentation. I'll do that here, and send a separate PR to the v1.14 branch with the following changes as well:

- install west==0.5.8, making note that later versions may work, but it's not guaranteed
- `west init --mr v1.14.1` (since we've already cut v1.14.0, I guess that PR will go out with the next point release)

Reply:
Thanks, I've already reviewed the separate PR